app/assets/javascripts/tooltips.js

```diff
@@ -0,0 +1,13 @@
+jQuery(function($) {
+  var tooltipElements = $('.advanced-tooltip');
+  var input = $('input[id^="new_password"]');
+  if(tooltipElements.length){
+    tooltipElements.each(function createTooltip() {
+      input.focusin(function () {
+        $(this).parent().parent().addClass('tooltip-visible');
```

Comment:
again, awfully specific to the use case - might be confusing for appliance fo other elements.